etc/kayobe/ansible/filter_plugins/filters.py

```diff
@@ -0,0 +1,57 @@
+#!usr/bin/env python3
+
+from ansible.errors import AnsibleFilterError
+# from ansible.plugins.filter.core imprt FilterModule
+
+class FilterModule(object):
+    def filters(self):
+        return {
+            'group_hostvars_by_var':
+            self.group_hostvars_by_var,
+            'group_hostvars_by_host':
+            self.group_hostvars_by_host
+        }
+
+    def group_hostvars_by_var(self, hostvars, var, stdout=None):
```

Comment:
Could you add [docstrings](https://peps.python.org/pep-0257/#multi-line-docstrings) for these filters?